library/analytics/src/main/java/org/cru/godtools/analytics/model/AnalyticsActionEvent.kt

```diff
@@ -26,8 +26,8 @@ open class AnalyticsActionEvent(
     open val firebaseEventName get() = action
     open val firebaseParams
         get() = Bundle().apply {
-            adobeAttributes?.forEach { putString(it.key.sanitizeAdobeNameForFirebase(), it.value?.toString()) }
-        }
+        adobeAttributes?.forEach { putString(it.key.sanitizeAdobeNameForFirebase(), it.value?.toString()) }
+    }
```

Comment:
I'm kind of surprised that this didn't trigger a ktlint violation for improper indention

Reply:
Yeah, I thought it didn't look right but ktlint didn't yell at me. I went ahead and fixed it though.